From 8e093e18406f6781a95bd49f6da5946562cb1a68 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marc-Andr=C3=A9=20Lureau?= <marcandre.lureau@gmail.com>
Date: Tue, 22 Jul 2014 13:22:02 +0200
Subject: [PATCH] virt-viewer.xml: remove zoom-{in,out} accelerators

The XML menu accelerators conflict with accelerator set in the code and
prevent using them. This is a regression from d29fc63d.
---
 src/virt-viewer.xml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/virt-viewer.xml b/src/virt-viewer.xml
index fd8295f..e2b836c 100644
--- a/src/virt-viewer.xml
+++ b/src/virt-viewer.xml
@@ -134,7 +134,6 @@
                                 <property name="use_action_appearance">False</property>
                                 <property name="use_underline">True</property>
                                 <property name="use_stock">True</property>
-                                <accelerator key="plus" signal="activate" modifiers="GDK_CONTROL_MASK"/>
                                 <signal name="activate" handler="virt_viewer_window_menu_view_zoom_in" swapped="no"/>
                               </object>
                             </child>
@@ -147,7 +146,6 @@
                                 <property name="use_action_appearance">False</property>
                                 <property name="use_underline">True</property>
                                 <property name="use_stock">True</property>
-                                <accelerator key="minus" signal="activate" modifiers="GDK_CONTROL_MASK"/>
                                 <signal name="activate" handler="virt_viewer_window_menu_view_zoom_out" swapped="no"/>
                               </object>
                             </child>
